src/Type/UnionIterableType.php

```diff
@@ -22,6 +22,20 @@ public function __construct(
 	}
 
 	/**
+	 * @return string|null
+	 */
+	public function getClass()
+	{
+		foreach ($this->otherTypes as $otherType) {
+			if ($otherType->getClass() !== null) {
+				return $otherType->getClass();
```

Comment:
I'd like it to return null if there are multiple other types with `getClass() !== null`, not the first one.